Sources/SwiftParserDiagnostics/ParserDiagnosticMessages.swift

```diff
@@ -518,6 +518,23 @@ public struct UnknownDirectiveError: ParserError {
   }
 }
 
+public struct UnknownParameterError: ParserError {
+  public let parameter: TokenSyntax
+  public let validParameters: [TokenSyntax]
+
+  public var message: String {
+    var message = "unknown parameter '\(parameter.description)'"
```

Comment:
Should this be `parameter.text` so that we don’t emit any trivia inside the single quotes?